style.css

```diff
@@ -0,0 +1,667 @@
+  z-index: 3;
+}
+
+.close-ham img {
+  margin: 1.5rem 0 0 1.5rem;
+}
+
+.nav-link a {
+  color: beige;
+  font-size: 22px;
+}
+
+.hamburg {
+  display: flex;
+  flex-direction: row;
+  align-self: flex-start;
+  width: 2rem;
+  padding: 0.2rem 0 0 1rem;
+  position: fixed;
+}
+
+.write-up {
+  background-image: url(img/main.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.showcase {
+  padding: 4rem 2rem 0 2rem;
+}
+
+.desc {
+  color: #ec5242;
+  font-weight: 700;
+  font-size: 35px;
+  max-width: 45rem;
+}
+
+.para {
+  background-color: rgba(105, 105, 105, 0.123);
+  width: 100%;
+  margin-top: 2rem;
+  padding: 1rem 2rem;
+}
+
+.date {
+  padding-top: 2rem;
+}
+
+.para1 {
+  padding-top: 0.5rem;
+}
+
+.logo {
+  display: none;
+}
+
+.img-logo {
+  max-width: 20%;
+  align-self: center;
+  padding: 1rem 0 1rem 1rem;
+}
+
+.program {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  background-color: #262930;
+  color: #fff;
+  margin-top: 2rem;
+}
+
+.pro p {
+  font-size: 0.8rem;
+  margin: 0 1rem 0.5rem 1rem;
+  width: 50%;
+}
+
+.pro h5 {
+  width: 15%;
+  padding-left: 1rem;
+  color: #e85242;
+}
+
+.join-us {
+  padding: 2rem 4rem;
+  background-color: #ec5242;
+  margin: 1rem 0;
+}
+
+.pro {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-top: 0.5rem;
+  background-color: dimgrey;
+}
+
+.speakers {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  background-color: white;
+  max-width: 100%;
+  margin: 0;
+  height: 100%;
+  padding: 0 0 2rem 2rem;
+}
+
+.speakers h2 {
+  padding-top: 1rem;
+  text-align: center;
+}
+
+.speaker-hr {
+  background-color: #3c3f45;
+}
+
+.speaker2-hr {
+  font-weight: bold;
+  width: 3rem;
+  border: 1px solid red;
+}
+
+.speaker1-hr {
+  font-weight: bold;
+  width: 3rem;
+  margin-top: 1rem;
+  border: 1px solid red;
+}
+
+.speaker-group img {
+  display: flex;
+  align-items: center;
+  max-width: 50%;
+  height: 30%;
+  margin: 0 0.5rem 1rem 0;
+}
+
+.speaker-group {
+  display: flex;
+  align-items: center;
+}
+
+.speaker-desc-holder {
+  margin-right: 1rem;
+  max-width: 50%;
+}
+
+.speaker-name {
+  font-size: 1rem;
+  font-weight: 600;
+  color: #535353;
+}
+
+.speaker-job {
+  font-size: 0.8rem;
+  color: #e85242;
+  font-style: italic;
+}
+
+.speaker-desc {
+  font-size: 0.8rem;
+}
+
+.program-text {
+  padding: 1.5rem 0;
+}
+
+.footer1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+  color: #fff;
+  padding: 2.5rem;
+  max-width: 100%;
+  height: auto;
+}
+
+.footer-logos {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-logo img {
+  width: 10rem;
+}
+
+.footer-logos img {
+  margin: 0 0.5rem;
+}
+
+.footer2-img img {
+  max-width: 40%;
+}
+
+.footer-about {
```

Comment:
- [ ] Mobile Size: Please resize the "footer" of "About page"  Mobile size, and make it look like the one on "Home page" (Mobile size). In doing so, I suggest that you reduce the "padding" in ".footer-about"